### DISCUSSION TITLE
[Q&A] Glpi_agent : Agent name in Inventory - custom ?

### DISCUSSION THREAD

**Discussion by Chico0008** [glpi-project/glpi-agent#319] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hi

when a new agent come to my inventory on my server, the agent name is always : host-date-time
is there a way to set the name to always be Only the hostname, or hostname.fqdn instead ?

**Comment by g-bougard** [comment]
Hi @Chico0008 
no, this is not possible and this is a bad idea and you should not care about agent name as this is just a reference to identify an agent in a given context.
